wikimedia/wikipedia-ios
# Updates localized string keys

**Phabricator:** N/A

### Notes
* Updates two identical localized string keys (talk-pages-topic-compose-close-confirmation-discard) representing different strings. I thought it should have broken our build, but somehow it slipped through a while ago

### Test Steps
1. Build the app, and make sure no related errors or warnings occur.
2. Start drafting and discard a topic on a talk page. Confirm the alert has the correct title.
3. Repeat the above for a talk page reply.
